### PR TITLE
Contrib: Add docs on the rustbot ready command

### PR DESCRIPTION
**Pull request by ehuss**
This adds some brief docs on the use of `@rustbot` [shortcuts](https://github.com/rust-lang/triagebot/wiki/Shortcuts).

### CODE REVIEW

File: src/doc/contrib/src/process/working-on-cargo.md [hunk 1]
```diff
@@ -122,6 +122,21 @@ typically finish in under 30 minutes.
 The reviewer might point out changes deemed necessary. Large or tricky changes
 may require several passes of review and changes.
 
+### Status labeling
+
+PRs will get marked with [labels] like [`S-waiting-on-review`] or [`S-waiting-on-author`] to indicate their status.
+The [`@rustbot`] bot can be used by anyone to adjust the labels.
+If a PR gets marked as `S-waiting-on-author`, and you have pushed new changes that you would like to be reviewed, you can write a comment on the PR with the text `@rustbot ready`.
```
Comment (weihanglo) [inline src/doc/contrib/src/process/working-on-cargo.md:129]: Not a big deal but these lines are inconsistent in terms of 80 columns convention in this file. Do you wanna update?

**Review by weihanglo — APPROVED** [review]

File: src/doc/contrib/src/process/working-on-cargo.md (hunk 1 above)
Comment (ehuss) [reply src/doc/contrib/src/process/working-on-cargo.md:129]: Some time ago I became a convert to [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/). I find that diffs involving text that has been line-wrapped can be difficult to read. Large amounts of the diff are related to the change in line-wrapping and not the actual content. Conversely, if an entire paragraph is *not* linewrapped, and there is a small change to the text, the diff would show a large change that can be difficult to spot. To combat those issues, semantic linefeeds strike a balance by only line-wrapping on sentences (and sometimes other punctuation like commas). mdBook markdown doesn't care how it is wrapped, and the diffs tend to be easier to view.

For new content I've been writing, I typically switch to semantic linefeeds.
Swapping out the old content can be a hassle, so I normally don't swap it out proactively.

Comment (weihanglo) [reply src/doc/contrib/src/process/working-on-cargo.md:129]: Good to know that! It seems like a good choice. I'll follow the rule as well.

**Pull request merged**